libvroom_examples/libvroom.cpp

```diff
@@ -211,8 +211,8 @@ void run_example_with_osrm() {
   // - jobs 5 and 6 can be served by either one of the vehicles
 
   // Solve!
-  auto sol = problem_instance.solve(5,  // Exploration level.
-                                    4); // Use 4 threads.
+  auto sol = problem_instance.solve(vroom::DEFAULT_EXPLORATION_LEVEL,  // Exploration level.
```

Comment:
Nit: this looks like a long line which `clang-format` would complain about. Can you run the [formatting script](https://github.com/VROOM-Project/vroom/blob/master/CONTRIBUTING.md#automatic-formatting)?

Reply:
Sure. I forgot about it.